core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
A couple of questions :
* why should these not contain the messages? If you don't include these, you would need to go to the database every time you need them.
* the message destination is the container of the messages. With your current design, you're implementing a relational  model and need to take care of deleting every message individually.
* Why do you not like that save on the destination repository?  Afraid of the overhead?  JPA keeps track of the changes and won't safe things that aren't changed